### ISSUE TITLE
Upload earlier releases

### ISSUE THREAD

**Issue by marcelstoer** [espressif/ESP8266_NONOS_SDK#30]
Earlier SDKs were, and still are, hosted on the Espressif BBS (e.g. http://bbs.espressif.com/download/file.php?id=1469). Unfortunately, BBS isn't as reliable as GitHub - it's currently down once again.
It'd be helpful if earlier releases were hosted here as well.

**Comment by russell** [comment]
Yep, It would be super fantastic if we could depend on this repo for future release zip files?

I did see that there are some files hosted here http://espressif.com/en/products/software/esp-sdk/resource  but are those URLs stable?  I raised the esd-open-sdk bug above because that project depends on files that are downloaded from the BBS, and it would be good if we had a more reliable hosting for the files other than the BBS.

Thanks :smile:
